package/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.23.3-alpine
+FROM nginx:stable-alpine3.19-perl
```

Comment:
Tags with `stable` will keep moving with newer releases. This might break something on a major release. Better to use some tags like nginx:1.27.0-alpine3.19-perl which would be more specific